python/lsst/obs/base/cameraMapper.py

```diff
@@ -395,8 +405,12 @@ def setMethods(suffix, mapImpl=None, bypassImpl=None, queryImpl=None):
                                [os.path.join(location.getStorage().root, p) for p in location.getLocations()])
                     # Metadata from FITS file
                     if subPolicy["storage"] == "FitsStorage":  # a FITS image
-                        setMethods("md", bypassImpl=lambda datasetType, pythonType, location, dataId:
-                                   readMetadata(location.getLocationsWithRoot()[0]))
+                        def getMetadata(datasetType, pythonType, location, dataId):
```

Comment:
I think this inline function needs a docstring. And the only argument that is used is `location`?

Similarly for the `getBBox`, `getMetadata`, and `getLen` below.

Reply:
Aren't they standard butler gen2 callbacks? I don't know what the args are but I assume butler gen2 requires they be there.

Reply:
I have no idea. Given the nature of gen2 documentation, I guess we can skip it?